packages/web-runtime/src/App.vue

```diff
@@ -159,6 +166,26 @@ export default defineComponent({
 
   methods: {
     ...mapActions(['fetchNotifications']),
+    ...mapActions(['showMessage', 'createModal', 'hideModal']),
+    ...mapActions('Files', ['copySelectedFiles', 'cutSelectedFiles', 'pasteSelectedFiles']),
```

Comment:
This is a no-no, sorry. The runtime should not have knowledge about apps and even needs to avoid imports from other apps. Otherwise we'll have hard or soft errors when web is being run without the files app (which is a valid use case!).
Could you please think about a way to register keyboard handlers in the runtime? I.e. the files app implements the keyboard handlers and uses an API from the runtime to register those keyboard handlers for certain keycodes and while being the active app?

Reply:
oh okay i see, yes should be doable